source/features/deep-reblame.tsx

```diff
@@ -68,25 +69,26 @@ async function redirectToBlameCommit(event: DelegateEvent<MouseEvent, HTMLAnchor
 
 	blameElement.blur(); // Hide tooltip after click, it’s shown on :focus
 
-	const prBlameCommit = await getPullRequestBlameCommit(prCommit, prNumber);
-	if (prBlameCommit) {
+	const [, currentFilename] = getCleanPathname().split(getReference()! + '/');
+
+	try {
+		const prBlameCommit = await getPullRequestBlameCommit(prCommit, prNumber, currentFilename);
 		const lineNumber = select('.js-line-number', blameHunk)!.textContent!;
 		const href = new URL(location.href.replace(getReference()!, prBlameCommit));
 		href.hash = 'L' + lineNumber;
 		location.href = String(href);
-		return;
-	}
-
-	if (blameElement instanceof HTMLAnchorElement) {
+	} catch (error) {
+		if (blameElement instanceof HTMLAnchorElement) {
 		// Restore the regular version link if there was one
-		blameElement.setAttribute('aria-label', 'View blame prior to this change.');
-		blameElement.classList.remove('rgh-deep-blame');
-		spinner.replaceWith(versionIcon());
-	} else {
-		spinner.remove();
-	}
+			blameElement.setAttribute('aria-label', 'View blame prior to this change.');
+			blameElement.classList.remove('rgh-deep-blame');
+			spinner.replaceWith(versionIcon());
+		} else {
+			spinner.remove();
+		}
```

Comment:
We should probably drop this. Buttons shouldn't disappear after being clicked. If the user missed the message they can click again and re-read it. It'll be handled by `mem` anyway